apps/app/cosmos.tf

```diff
@@ -34,3 +34,33 @@ resource "azurerm_cosmosdb_sql_role_assignment" "cosmos_access" {
   principal_id        = each.value
   scope               = "${data.azurerm_cosmosdb_account.state_store.id}/dbs/${azurerm_cosmosdb_sql_database.app.name}"
 }
+
+resource "azurerm_cosmosdb_sql_container" "example" {
+  name                  = "${var.app_id}-container"
+  resource_group_name   = var.resource_group_name
+  account_name          = var.cosmos_account_name
+  database_name         = azurerm_cosmosdb_sql_database.app.name
+  partition_key_path    = "/definition/id"
```

Comment:
think should just be `/id`

Reply:
Done ad608ff1cab05bf96cc5e42a0833e150468ebf03